src/markd/parsers/block.cr

```diff
@@ -210,7 +210,10 @@ module Markd::Parser
       container_parent = container.parent?
 
       container.open = false
-      container.source_pos[1] = [line_number, @last_line_length]
+      container.source_pos = {
+        {container.source_pos[0][0], container.source_pos[0][1]},
```

Comment:
```suggestion
        container.source_pos[0],
```

Reply:
Oh, well yeah, that's much simpler. Sorry I did this when it was quite late.